src/components/quote.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const Quote = () => (
+  <section>
+    <div>
+      <p>
+        Mathematics is not about numbers, equations, computations,
+        or algorithms: it is about understanding.
+        -William Paul Thurston
+      </p>
+    </div>
```

Comment:
[ OPTIONAL CHANGES ]

- [ ] It would be nice if you could add a padding-left to the style of this component so things get aligned well just as it is on the home page. The screenshot below is a guide as to how things should be aligned for good-looking design and consistency. 👍 

![image](https://user-images.githubusercontent.com/50941074/185602164-f0313727-efdf-4f71-a7eb-6c0c21f9a79d.png)